gmso/tests/files/restrained_benzene_ua.top

```diff
@@ -0,0 +1,76 @@
+; File Topology written by GMSO at 2022-10-25 13:20:26.887847
+
+[ defaults ]
+; nbfunc	comb-rule	gen-pairs		fudgeLJ	fudgeQQ
+1			2			no		0.0		0.0
+
+[ atomtypes ]
+; name		at.num	mass		charge		ptype	sigma	epsilon
+CH_sp2      6       13.01900     0.00000	A        0.36950     0.41988
+
+[ moleculetype ]
+; name	nrexcl
+Compound	3
+[ atoms ]
+; nr	type	resnr	residue		atom	cgnr	charge	mass
+1       CH_sp2      1       Compound    C       1        0.00000    13.01900
+2       CH_sp2      1       Compound    C       1        0.00000    13.01900
+3       CH_sp2      1       Compound    C       1        0.00000    13.01900
+4       CH_sp2      1       Compound    C       1        0.00000    13.01900
+5       CH_sp2      1       Compound    C       1        0.00000    13.01900
+6       CH_sp2      1       Compound    C       1        0.00000    13.01900
+
+[ bonds ]
+; ai	aj	funct	c0	c1
+2       1       1           0.14000        0.00000
+6       1       1           0.14000        0.00000
+3       2       1           0.14000        0.00000
+4       3       1           0.14000        0.00000
+5       4       1           0.14000        0.00000
+6       5       1           0.14000        0.00000
+
+[ angles ]
+; ai	aj	ak	funct	c0	c1
+2       1       6       1         120.00000        0.10000
+1       6       5       1         120.00000        0.10000
+4       5       6       1         120.00000        0.10000
+1       2       3       1         120.00000        0.10000
+3       4       5       1         120.00000        0.10000
+2       3       4       1         120.00000        0.10000
+
+[ angle_restraints ]
+; ai	aj	ai	ak	funct	theta_eq	k	multiplicity
```

Comment:
I don't know if this header needs the multiplicity.  can we varify this?

Reply:
There is a multiplicity in the angle_restraints per gromacs documentation (https://manual.gromacs.org/current/reference-manual/topologies/topology-file-formats.html, search for angle_restraints)

Reply:
OK.  I see what you mean, see the above comment